formats/label-to-vector-plugin/src/polus/plugins/formats/label_to_vector/__main__.py

```diff
@@ -0,0 +1,228 @@
+"""CLI for the label-to-vector-plugin."""
+
+import concurrent.futures
+import logging
+import pathlib
+import typing
+
+import bfio
+import filepattern
+import numpy
+import tqdm
+import typer
+import zarr
+from polus.plugins.formats.label_to_vector.dynamics.label_to_vector import convert
+from polus.plugins.formats.label_to_vector.utils import constants
+from polus.plugins.formats.label_to_vector.utils import helpers
+
+logging.basicConfig(
+    format="%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s",
+    datefmt="%d-%b-%y %H:%M:%S",
+)
+logger = helpers.make_logger("polus.plugins.formats.label_to_vector")
+
+app = typer.Typer()
+
+
+def flow_thread(
+    file_name: pathlib.Path,
+    zarr_path: pathlib.Path,
+    coordinates: tuple[int, int, typing.Optional[int]],
+) -> bool:
+    """Calculates the flows on a tile and saves it to the zarr file.
+
+    This is designed to run in a thread.
+
+    Args:
+        file_name: The file to read the tile from.
+        zarr_path: The zarr file to save the tile to.
+        coordinates: The coordinates of the tile to read.
+
+    Returns:
+        True if the tile was successfully processed.
+    """
+    x, y, z = coordinates
+    ndims = 2 if z is None else 3
+    z = 0 if z is None else z
+
+    # Load the data
+    with bfio.BioReader(file_name) as reader:
+        x_shape, y_shape, z_shape = reader.X, reader.Y, reader.Z
+
+        x_min = max(0, x - constants.TILE_OVERLAP)
```

Comment:
Shouldn't tile overlap be a parameter since it seems itmay need to be adjusted per application?